src/Perspex.SceneGraph/Animation/CrossFade.cs

```diff
@@ -8,12 +8,29 @@
 
 namespace Perspex.Animation
 {
+    // TODO: Perhaps we want a common base for Transitions with Duration so that we can 
+    // consolidate the property declarations, etc
+    //
+
     /// <summary>
     /// Defines a cross-fade animation between two <see cref="IVisual"/>s.
     /// </summary>
-    public class CrossFade : IPageTransition
+    public class CrossFade : PerspexObject, IPageTransition
```

Comment:
Yeah, PerspexProperties can only be added to a `PerspexObject`, but what i was asking is do these properties need to be PerspexProperties? Wouldn't a normal property be ok? Or do you imagine that these properties would be bound?